src/nfs/nfs_client_impl.cpp

```diff
@@ -567,12 +569,13 @@ void nfs_client_impl::register_cli_commands()
                 std::string result("OK");
 
                 if (args.empty()) {
-                    return std::to_string(FLAGS_max_copy_rate_megabytes);
```

Comment:
Does it make sense that change `FLAGS_max_copy_rate_megabytes` to `current_max_copy_rate_megabytes` in this pull request?

Reply:
`args.empty()` means query current value but default value